src/components/Home/Categories.jsx

```diff
@@ -0,0 +1,48 @@
+import hM from 'assets/h-m.jpg'
+import hS from 'assets/h-s.jpg'
+import nbaM from 'assets/nba-m.jpg'
+import nbaS from 'assets/nba-s.jpg'
+import sneakM from 'assets/sneak-m.jpg'
+import sneakS from 'assets/sneak-s.jpg'
+import styles from './home.module.css'
+
+const Categories = () => {
+
+    return (
+        <section className={`grid grid-maxcols-3 ${styles.categoryGrid} mg-top-md mg-btm-md`}>
```

Comment:
You can try classnames npm package that can help you write clean with multiple classes intead of using template literals

Reply:
Thanks. I will surely look into it.